molgenis-data-excel/src/main/java/org/molgenis/data/excel/ExcelEntity.java

```diff
@@ -44,17 +47,39 @@ public ExcelEntity(Row row, Map<String, Integer> colNamesMap, List<CellProcessor
 	@Override
 	public Object get(String attributeName)
 	{
-		Integer col = colNamesMap.get(attributeName);
-		if (col != null)
+		if (cachedValueMap == null)
+		{
+			cachedValueMap = new CaseInsensitiveLinkedHashMap<>();
+		}
+
+		Object value;
```

Comment:
Ok, for not forgetting you can use final like this:
final Object value;
Then the code do not compile if it is not instantiated.